static/styles/portico/portico.css

```diff
@@ -433,6 +433,61 @@ img.screenshot {
     #portico-area {
         margin-left: 1.5em;
     }
+
+    /* To reveal the whole page for printing we should show the content of the body,
+    since printing dialog can't scroll content inside nested containers (simplebar) as a user. */
+    body.noscroll {
+        overflow: auto;
+        position: static;
+        width: 100%;
+    }
+
+    /* In a browser markdown container calculates the height and the width close to the browsers viewport,
+    but for printing we need the content to expand its size as usual block element. */
+    .help .markdown {
+        height: inherit;
+        width: 100%;
+        position: static;
+    }
+
+    /* Enabling header only on the first printing page. */
```

Comment:
This is very much a minor style nit, but we prefer the imperative form, i.e. "Display header only on the first printed page."